src/grt/src/fastroute/include/FastRoute.h

```diff
@@ -99,6 +99,25 @@ struct parent3D
   int x, y;
 };
 
+struct CostParams
+{
+  const float logis_cof;
+  const float cost_height;
+  const int slope;
+  const int cost_type;
+
+  CostParams(const float logis_cof,
+             const float cost_height,
+             const int slope,
+             const int cost_type)
+      : logis_cof(logis_cof),
+        cost_height(cost_height),
+        slope(slope),
+        cost_type(cost_type)
```

Comment:
It would be nice to understand what each cost type means and change it from `int` to an `enum class`, with descritive names. Today we only compare with numbers, which is very unclear.